diff --git a/node_modules/@geist-ui/react/esm/styles/themes/dark.js b/node_modules/@geist-ui/react/esm/styles/themes/dark.js
index 9ce6920..3b69156 100644
--- a/node_modules/@geist-ui/react/esm/styles/themes/dark.js
+++ b/node_modules/@geist-ui/react/esm/styles/themes/dark.js
@@ -1,7 +1,7 @@
 import { defaultFont, defaultBreakpoints, defaultLayout } from './shared';
 export var palette = {
   /* eslint-disable @typescript-eslint/camelcase */
-  accents_1: '#111',
+  accents_1: '#2f3437',
   accents_2: '#333',
   accents_3: '#444',
   accents_4: '#666',
@@ -11,7 +11,7 @@ export var palette = {
   accents_8: '#fafafa',
 
   /* eslint-enable camelcase */
-  background: '#000',
+  background: '#171717',
   foreground: '#fff',
   selection: '#f81ce5',
   secondary: '#888',
